l10n_ro_stock_account_date/models/stock_move.py

```diff
@@ -42,6 +42,13 @@ def _get_price_unit(self):
         # Update price unit for purchases in different currencies with the
         # reception date.
         if self.is_l10n_ro_record:
+            if (
+                self.origin_returned_move_id
+                and self.origin_returned_move_id.sudo().stock_valuation_layer_ids
+            ):
+                # in acest caz trebuie sa se execute din l10n_ro_stock_account
+                return super()._get_price_unit()
+
             if self.picking_id.date and self.purchase_line_id:
```

Comment:
@feketemihai 
aici in l10n_ro_stock_account_date se face override complet fata de cea din l10n_ro_stock_account
dar daca este return, trebuie sa se execute din l10n_ro_stock_account